2024-03-15 07:28:12 - Traceback (most recent call last):
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_app.py", line 500, in setSock
    dispatcher.read(self.sock.sock, read, check)
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_app.py", line 112, in read
    if not read_callback():
           ^^^^^^^^^^^^^^^
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_app.py", line 523, in read
    raise e
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_app.py", line 515, in read
    op_code, frame = self.sock.recv_data_frame(True)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_core.py", line 437, in recv_data_frame
    frame = self.recv_frame()
            ^^^^^^^^^^^^^^^^^
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_core.py", line 478, in recv_frame
    return self.frame_buffer.recv_frame()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_abnf.py", line 363, in recv_frame
    self.recv_header()
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_abnf.py", line 319, in recv_header
    header = self.recv_strict(2)
             ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_abnf.py", line 398, in recv_strict
    bytes_ = self.recv(min(16384, shortage))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_core.py", line 563, in _recv
    return recv(self.sock, bufsize)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\yanni\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_socket.py", line 129, in recv
    raise WebSocketConnectionClosedException("Connection to remote host was lost.")
websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.
